client/state/login/actions.js

```diff
@@ -150,6 +159,17 @@ export const loginUser = ( usernameOrEmail, password, rememberMe, redirectTo ) =
 				rememberMe,
 				data: response.body && response.body.data,
 			} );
+
+			if ( get( response, 'body.data.two_step_notification_sent', null ) === 'sms' ) {
```

Comment:
I don't think we need to specify `null` here: if we omit this third parameter then `get()` will return `undefined` which won't be equal to `sms`.